Makefile

```diff
@@ -7,6 +7,13 @@ cppflags := -DDATADIR=$(DATADIR)
 
 sources := $(wildcard src/brogue/*.c) $(addprefix src/platform/,main.c platformdependent.c)
 
+ifeq ($(RELEASE),YES)
+	extra_version :=
+else
+	extra_version := $(shell tools/git-extra-version)
+endif
+cppflags += -DBROGUE_EXTRA_VERSION='$(extra_version)'
```

Comment:
The branch as-is fails to compile, I believe because we dropped the quotes around here.  The preprocessor is now inserting `the-extra-version`, not `"the-extra-version"`, which understandably results in syntax errors.

I'll grab it and fixup the new commits at the same time, one moment..

Reply:
Oh right yeah, because it needs to be a C literal, my bad

Reply:
..It's working for me locally now, but the would-be CI still needs the SDL fixes from the other branch.  Still, I think we're mergeable now.